.bldr.toml

```diff
@@ -632,6 +632,8 @@ plan_path = "kubernetes-scheduler"
 paths = [
   "kubernetes/*",
 ]
+[lastpass-cli]
+plan_path = "lastpath-cli"
```

Comment:
I think the plan path here is incorrect.

Reply:
It’s been corrected.